app/src/main/java/com/sdp/movemeet/Sport.java

```diff
@@ -5,5 +5,17 @@
     Soccer,
     Swimming,
     Tennis,
-    Badminton
+    Badminton,
```

Comment:
Wahoo, you added a lot of new sport activities! Do they all have their own icon?

Reply:
Sorry I initially didn't that you also committed a png for each of those new sport activities 👍

Reply:
Yes ! But I noticed that they all had different sizes, so I had to resize them to 24x24. Some of them lose in quality, so we might have to change these icons later on.